### PR TITLE
feat(): PostgreSQL example with snapshot triggered by Kafka event

### PR DESCRIPTION
**Pull request by pneff93**
see the conversation on LinkedIn: https://www.linkedin.com/feed/update/urn:li:activity:7115665658553507840?commentUrn=urn%3Ali%3Acomment%3A%28activity%3A7115665658553507840%2C7119767027791065090%29&replyUrn=urn%3Ali%3Acomment%3A%28activity%3A7115665658553507840%2C7120710931172458496%29&dashCommentUrn=urn%3Ali%3Afsd_comment%3A%287119767027791065090%2Curn%3Ali%3Aactivity%3A7115665658553507840%29&dashReplyUrn=urn%3Ali%3Afsd_comment%3A%287120710931172458496%2Curn%3Ali%3Aactivity%3A7115665658553507840%29

### CODE REVIEW

File: postrges-kafka-signal/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# PostgreSQL Snapshot via Kafka event 
+
+With the release of Debezium version 2.3, table snapshots cannot just 
+be triggered by an insert into signal table but also by a message into a signal Kafka topic.
```
Comment (mfvitale) [inline postrges-kafka-signal/README.md:4]:
```suggestion
With the release of Debezium version 2.3, incremental snapshot can be [triggered in a different way](https://debezium.io/documentation/reference/2.3/configuration/signalling.html) and not only via an insert into signal table. Also Kafka signal, that was available only for MySQL with GTIDs enabled, is now an option for all connectors.
```

File: postrges-kafka-signal/connector.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    "topic.prefix": "test",
+
+    "table.include.list" : "public.characters",
+    "signal.data.collection": "public.debezium_signal",
+    "signal.enabled.channels": "source,kafka",
+    "signal.kafka.topic": "signal-topic",
+    "signal.kafka.bootstrap.servers": "broker:29092",
+
+    "_comment:": "remove _ to use ExtractNewRecordState smt",
```
Comment (mfvitale) [inline postrges-kafka-signal/connector.json:20]: Can this be removed?

Comment (pneff93) [reply postrges-kafka-signal/connector.json:20]: yes

**Comment by mfvitale** [comment]
@pneff93 Thanks for your contribution. Overall LGTM, I just leaved minor comments.

File: postrges-kafka-signal/connector.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    "_transforms": "unwrap",
+    "_transforms.unwrap.type": "io.debezium.transforms.ExtractNewRecordState"
```
Comment (mfvitale) [inline postrges-kafka-signal/connector.json:22]:
```suggestion
```

**Comment by pneff93** [comment]
fixed your suggestions and squashed commits for simpler history

**Comment by mfvitale** [comment]
@pneff93 Applied. Thanks a lot.

**Pull request merged**